platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/wires/input/BindableInputWire.java

```diff
@@ -121,6 +121,7 @@ public BindableInputWire<IN, OUT> bind(@NonNull final Function<IN, OUT> handler)
                 taskSchedulerInput.forward(output);
             }
         });
+        model.registerInputWireBinding(taskSchedulerName, getName());
```

Comment:
@cody-littley this seems to have been a gap in the wiring code

Reply:
Yes, I think this was a bug. My unit tests for modeling use the other variant of the bind function.